### PR TITLE
DOCS-2849 Single Source the Mobile Vitals Doc Page

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Single sources the Mobile Vitals page from the Android SDK repo.

### Motivation

DOCS-2849

### Preview

https://docs-staging.datadoghq.com/alai97/rum-android-vitals-single-sourcing/real_user_monitoring/android/mobile_vitals

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -366,6 +366,24 @@
...
               text: "Explore Datadog RUM"
+    - action: pull-and-push-file
+      branch: master
+      globs:
+      - 'docs/troubleshooting_android.md'
+      options:
+        dest_path: '/real_user_monitoring/android/'
+        file_name: 'troubleshooting.md'
+        front_matters:
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/troubleshooting_android.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:377]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/android-monitoring-mobile-vitals/docs/troubleshooting_android.md"]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -366,6 +366,24 @@
             - link: "/real_user_monitoring"
               tag: "Homepage"
               text: "Explore Datadog RUM"
+    - action: pull-and-push-file
+      branch: master
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:370]:
```suggestion
      branch: alai97/android-monitoring-mobile-vitals
```

File: local/bin/py/build/configurations/pull_config.yaml [hunk 3]
```diff
@@ -365,6 +365,24 @@
...
               text: "Explore Datadog RUM"
+    - action: pull-and-push-file
+      branch: master
+      globs:
+      - 'docs/troubleshooting_android.md'
+      options:
+        dest_path: '/real_user_monitoring/android/'
+        file_name: 'troubleshooting.md'
+        front_matters:
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/troubleshooting_android.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config.yaml:376]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/android-monitoring-mobile-vitals/docs/troubleshooting_android.md"]
```

File: local/bin/py/build/configurations/pull_config.yaml [hunk 4]
```diff
@@ -365,6 +365,24 @@
             - link: "/real_user_monitoring"
               tag: "Homepage"
               text: "Explore Datadog RUM"
+    - action: pull-and-push-file
+      branch: master
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config.yaml:369]:
```suggestion
      branch: alai97/android-monitoring-mobile-vitals
```

**Review by alai97 — COMMENTED** [review]
Changing these to my branch to generate a preview in GitLab.

**Review by priyanshi-gupta — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 5]
```diff
@@ -331,23 +331,23 @@
               tag: "Homepage"
               text: "Explore Datadog RUM"
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/android-monitoring-mobile-vitals
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:334]:
```suggestion
      branch: master
```

File: config/_default/menus/menus.en.yaml [hunk 6]
```diff
@@ -2671,16 +2671,21 @@ main:
     parent: rum_android
     identifier: rum_android_advanced
     weight: 202
-  - name: Troubleshooting
-    url: real_user_monitoring/android/troubleshooting/
+  - name: Mobile Vitals
+    url: real_user_monitoring/android/mobile_vitals/
     parent: rum_android
-    identifier: rum_troubleshooting_android
+    identifier: rum_troubleshooting_android_mobile_vitals
```
Comment (alai97) [inline config/_default/menus/menus.en.yaml:2677]:
```suggestion
    identifier: rum_android_mobile_vitals
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 7]
```diff
@@ -331,23 +331,23 @@
...
-      - 'docs/troubleshooting_android.md'
+      - 'docs/rum_mobile_vitals.md'
       options:
         dest_path: '/real_user_monitoring/android/'
-        file_name: 'troubleshooting.md'
+        file_name: 'mobile_vitals.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/troubleshooting_android.md"]
-          title: Troubleshooting
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/android-monitoring-mobile-vitals/docs/rum_mobile_vitals.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:341]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_mobile_vitals.md"]
```

File: local/bin/py/build/configurations/pull_config.yaml [hunk 8]
```diff
@@ -330,23 +330,23 @@
...
-      - 'docs/troubleshooting_android.md'
+      - 'docs/rum_mobile_vitals.md'
       options:
         dest_path: '/real_user_monitoring/android/'
-        file_name: 'troubleshooting.md'
+        file_name: 'mobile_vitals.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/troubleshooting_android.md"]
-          title: Troubleshooting
+          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/alai97/android-monitoring-mobile-vitals/docs/rum_mobile_vitals.md"]
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config.yaml:340]:
```suggestion
          dependencies: ["https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_mobile_vitals.md"]
```

File: local/bin/py/build/configurations/pull_config.yaml [hunk 9]
```diff
@@ -330,23 +330,23 @@
               tag: "Homepage"
               text: "Explore Datadog RUM"
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/android-monitoring-mobile-vitals
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config.yaml:333]:
```suggestion
      branch: master
```

**Review by alai97 — COMMENTED** [review]
Switches staging branch to master branch of dd-sdk-android.

**Pull request merged**